backend/src/main/java/com/selflearn/backend/user/UserController.java

```diff
@@ -1,4 +1,21 @@
 package com.selflearn.backend.user;
 
+import com.selflearn.backend.user.models.User;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/users")
```

Comment:
consider add version to the url path
```suggestion
@RequestMapping("/v1/users")
```